app/stac_api/validators.py

```diff
@@ -168,6 +171,46 @@ def validate_link_rel(value):
         )
 
 
+def validate_text_to_geometry(text_geometry):
+    '''
+    https://jira.swisstopo.ch/browse/BGDIINF_SB-1650
+    A validator function that tests, if a text can be transformed to a geometry.
+    The text is either a bbox or WKT.
+
+    an extent in either WGS84 or LV95, in the form "(xmin, ymin, xmax, ymax)" where x is easting
+    a WKT defintion of a polygon in the form POLYGON ((30 10, 40 40, 20 40, 10 20, 30 10))
+
+    Args:
+        text_geometry: The text to be transformed into a geometry
+    Returns:
+        the text_geometry
+    Raises:
+        ValidationError: About that the text_geometry is not valid
+    '''
+    if not text_geometry:
+        message = "The geometry is a required field: %s"
+        params = {'error': 'empty field'}
+        logger.error(message, params)
+        raise ValidationError(_(message), params=params, code='invalid') from None
+    # is the input WKT
+    try:
+        geos_geometry = GEOSGeometry(text_geometry)
+        validate_geometry(geos_geometry)
+        return geos_geometry
+    except (ValueError, ValidationError, IndexError, GDALException, GEOSException) as error:
+        message = "The text as WKT could not be transformed into a geometry: %s " % error
+    # is the input a bbox
+    try:
+        text_geometry = text_geometry.replace('(', '')
+        text_geometry = text_geometry.replace(')', '')
```

Comment:
Why this ? From the examples above the bbox should be `x1,y1,x2,y2`.

Reply:
in the ticket they (the client) wanted to put (x1,y1,x2,y,2) as input ... so with this we don't have to care ...

Reply:
Ok :-)